node/services/service.go

```diff
@@ -254,17 +289,13 @@ func FraudLifecycle(
 		return &fraud.ErrFraudExists{Proof: proofs}
 	case datastore.ErrNotFound:
 	}
-	err = start(startCtx)
-	if err != nil {
-		return err
-	}
 	// handle incoming Fraud Proofs
 	go fraud.OnProof(lifecycleCtx, fservice, p, func(fraud.Proof) {
 		if err := stop(lifecycleCtx); err != nil {
 			log.Error(err)
 		}
 	})
-	return nil
```

Comment:
If Start fails, we should not start a routine to handle proof for the unstarted service, so the previous code is correct.